src/code/z_message_PAL.c

```diff
@@ -3073,6 +3072,8 @@ void Message_Update(PlayState* play) {
 
 #if OOT_DEBUG
     if (BREG(0) != 0) {
+        static u16 D_80153D78 = 0;
+
         if (CHECK_BTN_ALL(input->press.button, BTN_DDOWN) && CHECK_BTN_ALL(input->cur.button, BTN_L)) {
```

Comment:
if we wanna just name this now, it looks like it can be `sMessageDebuggerTextboxCount`, or something..
it counts up by 1 every time the message debugger is used to display a textbox, and doesnt serve any purpose other than to print that number out to the console

Reply:
`sMessageDebuggerTextboxCountMod10`? that's a comically long name though